### PR TITLE
Add option to render A4 size

### PR DESCRIPTION
**Pull request by claudep**

### CODE REVIEW

File: qrbill/bill.py [hunk 1]
```diff
@@ -498,45 +540,52 @@ def add_header(text):
...
+    )
+
+
+def mm(val):
+    """Convert val (as mm, either number of '12mm' str) into user units."""
+    try:
+        val = float(val.rstrip('mm'))
+    except AttributeError:
+        pass
+    return round(val * MM_TO_UU, 2)
```
Comment (sbor23) [inline qrbill/bill.py:588]: I don't think it matters, but shouldn't we try to keep the precision instead of rounding?

**Comment by sbor23** [comment]
Can confirm, this looks much better!
Thanks a lot!

File: qrbill/bill.py (hunk 1 above)
Comment (claudep) [reply qrbill/bill.py:588]: We are talking about fraction of pixels here. Having `32.456733452`-type values doesn't make much sense, but admittedly when printing, rounding to 2 decimals is a bit too short. Maybe round to 4 or 5 decimals?

Comment (sbor23) [reply qrbill/bill.py:588]: Maybe 5 just for good measure :)

**Comment by claudep** [comment]
Many thanks for your contribution!

**Pull request merged**